task_10/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/reset.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <script type="text/javascript" src="js/config.js"></script>
+    <script type="text/javascript" src="js/helper.js"></script>
+    <script type="text/javascript" src="js/models/model.js"></script>
+    <script type="text/javascript" src="js/views/view.js"></script>
+    <script type="text/javascript" src="js/services/weather_service.js"></script>
+    <script type="text/javascript" src="js/services/storage_service.js"></script>
+    <script type="text/javascript" src="js/services/list_service.js"></script>
+    <script type="text/javascript" src="js/services/favorites_service.js"></script>
+    <script type="text/javascript" src="js/services/history_service.js"></script>
+    <script type="text/javascript" src="js/controllers/controller.js"></script>
+    <script type="text/javascript" src="js/app.js"></script>
+    <title>Weather App</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="flex-container">
+          <span>
+            <button class="btn" id="add-favorite-btn" title="Adds city to favorites">
+              <i class="fa fa-star" aria-hidden="true"></i>
+            </button>
+          </span>
+          <span class="loc-field">
+            <input class="btn wide" id="loc-field" type="text" placeholder="Enter location..." title="Enter location here">
+          </span>
+          <span class="base-units">
+            <select class="btn" id="base-units" title="Selects metric or imperial units">
+              <option value="metric">&deg;C, m/s</option>
+              <option value="imperial">&deg;F, mph</option>
+            </select>
+          </span>
+      </div>
+    </header>
+    <main>
+      <section>
+        <div class="flex-container current-day" id="current-day"></div>
```

Comment:
Looks strange. Why do you need nested container?

Reply:
Fixed.